src/App.css

```diff
@@ -1,107 +1,26 @@
 .App {
-  text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #b9dfed;
+    position:relative;
+}
+
+.Header {
+    position: fixed;
+    width: 100vw;
+    padding:0;
+    background-color: white;
+    max-height: 100px;
```

Comment:
If you get rid of the max-height, it will still look the same, and respond better with the window re-sizing (just allows the text to wrap around).

Reply:
got it!